src/data/finset/intervals.lean

```diff
@@ -29,8 +29,8 @@ namespace Ico
 @[simp] theorem to_finset (n m : ℕ) : (multiset.Ico n m).to_finset = Ico n m :=
 (multiset.to_finset_eq _).symm
 
-theorem image_add (n m k : ℕ) : (Ico n m).image ((+) k) = Ico (n + k) (m + k) :=
-by simp [image, multiset.Ico.map_add]
+theorem map_add (n m k : ℕ) : (Ico n m).map (add_right_embedding k) = Ico (n + k) (m + k) :=
+by simp [Ico, map, multiset.Ico.map_add, add_comm _ k]
```

Comment:
I think it's worth having both versions of the lemma here:
```suggestion
theorem image_add (n m k : ℕ) : (Ico n m).image ((+) k) = Ico (n + k) (m + k) :=
by simp [image, multiset.Ico.map_add]

theorem map_add (n m k : ℕ) : (Ico n m).map (add_right_embedding k) = Ico (n + k) (m + k) :=
by simp [Ico, map, multiset.Ico.map_add, add_comm _ k]
```

IMO both `multiset.Ico.map_add` and `finset.Ico.image_add` are backwards and should have `Ico (k + n) (k + m)` on their RHS.

Reply:
Agreed. Agreed. Those lemmas are heavily generalized in #7987 and `image_add` will be more general than your new lemma `map_add`.